week8/week8/ContentView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ContentView.swift
+//  Mod8JSON
+//
+//  Created by Nick Ryan on 6/8/24.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+  @StateObject private var universityStore = UniversityStore()
+  
+  
+  var body: some View {
+    NavigationStack {
+      VStack {
```

Comment:
You're not stacking anything here. If you don't have to stack view no need for a `VStack`